_content/av/knowing-the-mind_anandabodhi.md

```diff
@@ -0,0 +1,18 @@
+---
+title: "Knowing the Mind"
+authors:
+  - anandabodhi
+external_url: "https://dharmaseed.org/talks/40609/"
+drive_links:
+  - "https://drive.google.com/file/d/1ykYxlM4nnA9smL1qRfeA8U4rmbmmfoQw/view?usp=sharing"
+course: problems
+tags:
+  - satipatthana
+  - hindrances
+  - depression
+year: 2017
+month: may
+minutes: 42
+---
+
+Here, Ayya Anandabodhi delivers a talk on the importance of the satipaṭṭhāna practice, particularly it's dual role in mental health and awakening.
```

Comment:
Possessive 'its' has no apostrophe in English 😉  and "Here" is redundant.  And awakening and mental health are related, so I don't see how that's a "dual" role?